crates/bevy_wgpu/src/lib.rs

```diff
@@ -32,11 +32,39 @@ impl Plugin for WgpuPlugin {
 }
 
 pub fn wgpu_render_system(resources: &mut Resources) -> impl FnMut(&mut World, &mut Resources) {
-    let mut wgpu_renderer = pollster::block_on(WgpuRenderer::new());
+    let options = resources
+        .get_cloned::<WgpuOptions>()
+        .unwrap_or_else(|| WgpuOptions::new());
```

Comment:
Remove redundant closure.

```suggestion
        .unwrap_or_else(WgpuOptions::new);
```